maven-release-manager/src/main/java/org/apache/maven/shared/release/DefaultReleaseManager.java

```diff
@@ -560,15 +559,10 @@ private ReleaseDescriptorBuilder loadReleaseDescriptorBuilder( ReleaseDescriptor
     {
         try
         {
-            updateListener( listener, "verify-release-configuration", PHASE_START );
-            ReleaseDescriptorBuilder result = configStore.get().read( builder );
-            updateListener( listener, "verify-release-configuration", PHASE_END );
-            return result;
+            return configStore.get().read( builder );
         }
         catch ( ReleaseDescriptorStoreException e )
         {
-            updateListener( listener, e.getMessage(), ERROR );
```

Comment:
Why is this information not necessary anymore?

Reply:
same as previous: it's not a phase
and the error will be reported in the exception

Reply:
Agreed.